frontend/src/scenes/dashboard/DashboardItem.js

```diff
@@ -127,129 +128,150 @@ export function DashboardItem({
             <div className={`dashboard-item-container ${className}`}>
                 <div className="dashboard-item-header">
                     <div className="dashboard-item-title">
-                        <Link
-                            draggable={false}
-                            to={link}
-                            title={item.name}
-                            preventClick
-                            onClick={() => {
-                                if (!isDraggingRef.current) {
-                                    router.actions.push(link)
-                                }
-                            }}
-                        >
-                            {item.name}
-                        </Link>
+                        {inSharedMode ? (
+                            item.name
```

Comment:
There's still a "cursor:move" effect that shows up on the title, indicating it could be moved in shared mode, but dragging that will have no effect.